frontend/src/common/components/Organism/Header.vue

```diff
@@ -1,53 +1,55 @@
 <script setup lang="ts">
+import Logo from '@/common/assets/codedang_logo.png'
 import { useAuthStore } from '@/common/store/auth'
 import { OnClickOutside } from '@vueuse/components'
 import { ref } from 'vue'
-import { useRouter } from 'vue-router'
-import IconUser from '~icons/fa6-regular/user'
 import IconArrowRightFromBracket from '~icons/fa6-solid/arrow-right-from-bracket'
 import IconBars from '~icons/fa6-solid/bars'
 import IconSliders from '~icons/fa6-solid/sliders'
+import IconUser from '~icons/fa6-solid/user'
 import IconUserGear from '~icons/fa6-solid/user-gear'
 import Button from '../Atom/Button.vue'
 import ListItem from '../Atom/ListItem.vue'
-import SignatureLogo from '../Atom/SignatureLogo.vue'
 import Dropdown from '../Molecule/Dropdown.vue'
 import AuthModal from './AuthModal.vue'
 
 const auth = useAuthStore()
-const router = useRouter()
 const isMenuOpen = ref(false)
 const modalContent = ref<'login' | 'signup' | 'password' | 'close'>('close')
 </script>
 
 <template>
-  <OnClickOutside @trigger="isMenuOpen = false">
-    <header class="border-b-gray grid h-14 place-items-center border-b px-8">
-      <div class="flex w-full max-w-7xl items-center justify-between">
-        <RouterLink to="/">
-          <SignatureLogo
-            class="w-40 cursor-pointer active:opacity-40 md:hover:opacity-60"
-          />
-        </RouterLink>
-        <nav class="text-text-title hidden gap-4 md:flex">
-          <RouterLink
-            v-for="{ to, name } in [
-              { to: '/notice', name: 'Notice' },
-              { to: '/contest', name: 'Contest' },
-              { to: '/problem', name: 'Problem' },
-              { to: '/group', name: 'Group' }
-            ]"
-            :key="name"
-            class="cursor-pointer text-lg font-semibold hover:opacity-60 active:opacity-40"
-            :to="to"
-            :class="{
-              'text-green hover:opacity-70 active:opacity-50':
-                router.currentRoute.value.fullPath.startsWith(to)
-            }"
-          >
-            {{ name }}
+  <OnClickOutside
+    class="sticky top-0 z-[9999] bg-white"
+    @trigger="isMenuOpen = false"
+  >
+    <header class="border-b-gray grid h-20 place-items-center border-b px-8">
+      <div class="flex w-full max-w-7xl items-center justify-between gap-5">
+        <div class="flex items-center gap-32">
+          <RouterLink to="/">
+            <div class="flex items-center gap-1">
+              <img :src="Logo" alt="logo" width="45" />
+              <h1 class="text-2xl font-bold">Codedang</h1>
+            </div>
```

Comment:
`RouterLink`는 anchor element로 rendering돼서, div로 wrap하는 대신 `RouterLink`에 class 정의하는 게 좋을 것 같아요!

```suggestion
          <RouterLink to="/" class="flex items-center gap-1">
            <img :src="Logo" alt="logo" width="45" />
            <h1 class="text-2xl font-bold">Codedang</h1>
```

Reply:
그리고 hover effect도 있으면 좋을 것 같습니다!

Reply:
한 번 이펙트 넣어볼게요!

Reply:
로고가 있는 링크는 보통 이펙트는 넣치 않는게 best practice인 것 같아서 추가 안했습니다!

Reply:
보통 로고가 있는 링크는 hover effect 안 넣는게 best practice라서 안 넣었습니다!